microsoft/runbook/bare_metal.yml

```diff
@@ -0,0 +1,70 @@
+name: $(test_name)
```

Comment:
bare metal is highly customizable, so we cannot provide a sample runbook. Please move samples to document. We need to provide current supported components in documents.